lib/fizzy/parser_expr.cpp

```diff
@@ -153,14 +153,20 @@ inline void validate_branch_stack_height(
 }
 
 
-void push_branch_immediates(const ControlFrame& frame, bytes& immediates)
+void push_branch_immediates(const ControlFrame& frame, int branch_stack_height, bytes& immediates)
```

Comment:
Better rename parameters to be similar to variables on the caller side
```suggestion
void push_branch_immediates(const ControlFrame& branch_frame, int stack_height, bytes& immediates)
```